app/src/main/res/values/strings.xml

```diff
@@ -228,6 +230,11 @@ How to translate with transifex:
     <string name="send_email">Send email</string>
     <string name="create_issue">Create issue</string>
     <string name="nc_diagnose_flavor" translatable="false">Build flavor</string>
+    <string name="nc_test_push_button">"Test push notifications</string>
+    <string name="nc_test_results">Test results</string>
+    <string name="message_copied">Message copied</string>
+    <string name="nc_push_notification_message">Push notification is sent successfully. You should now receive
+        a notification on this device with the title \'Testing push notifications\'  </string>
```

Comment:
the trailing space on translation string, please read the do's and don'ts
https://docs.nextcloud.com/server/latest/developer_manual/basics/translations.html#dos-and-don-ts